conda_smithy/templates/azure-pipelines-linux.yml.tmpl

```
@@ -33,6 +33,14 @@ jobs:
          sudo docker image prune --all --force
          df -h
     displayName: Manage disk space
+{%- endif %}
+{%- if azure.linux_swap_size is defined %}
```

Per this comment ( https://github.com/conda-forge/conda-forge.github.io/pull/1988#discussion_r1275490253 ) maybe we can check for `0`(or `undefined`) like so?

```suggestion
{%- if (azure.linux_swap_size|str|default("0")).startswith(0) %}
```